plans/upstream-keylime-containers.fmf

```diff
@@ -0,0 +1,33 @@
+summary:
+  Tests used by Packit/TFT CI on Github to test upstream keylime containers
+
+environment+:
+  TPM_BINARY_MEASUREMENTS: /var/tmp/binary_bios_measurements
+  KEYLIME_RUST_CODE_COVERAGE: 1
```

Comment:
For containers we are not really measuring code coverage so we probably don't have to set this one.

Reply:
Thanks, I'll fix.